[2016-01-05 21:04:32,515] {main.py->handle_error:54} INFO - [97mINFO: [39mInside Main->handle_error | app.errorhandler( Exception )
[2016-01-05 21:04:32,518] {/Users/roporter/Desktop/Development/Python/myAPI/app/classes/requests.py->after_request:35} INFO - [97mINFO: [39mInside Main->returnResponse | app.after_request
[2016-01-05 21:04:32,591] {main.py->handle_error:54} INFO - [97mINFO: [39mInside Main->handle_error | app.errorhandler( Exception )
[2016-01-05 21:04:32,592] {/Users/roporter/Desktop/Development/Python/myAPI/app/classes/requests.py->after_request:35} INFO - [97mINFO: [39mInside Main->returnResponse | app.after_request
[2016-01-05 21:04:56,845] {/Users/roporter/Desktop/Development/Python/myAPI/app/classes/requests.py->after_request:35} INFO - [97mINFO: [39mInside Main->returnResponse | app.after_request
[2016-01-05 21:04:57,286] {/Users/roporter/Desktop/Development/Python/myAPI/app/classes/requests.py->after_request:35} INFO - [97mINFO: [39mInside Main->returnResponse | app.after_request
[2016-01-05 21:04:57,404] {main.py->handle_error:54} INFO - [97mINFO: [39mInside Main->handle_error | app.errorhandler( Exception )
[2016-01-05 21:04:57,405] {/Users/roporter/Desktop/Development/Python/myAPI/app/classes/requests.py->after_request:35} INFO - [97mINFO: [39mInside Main->returnResponse | app.after_request
[2016-01-05 21:04:57,527] {main.py->handle_error:54} INFO - [97mINFO: [39mInside Main->handle_error | app.errorhandler( Exception )
[2016-01-05 21:04:57,528] {/Users/roporter/Desktop/Development/Python/myAPI/app/classes/requests.py->after_request:35} INFO - [97mINFO: [39mInside Main->returnResponse | app.after_request
[2016-01-05 21:04:57,664] {main.py->handle_error:54} INFO - [97mINFO: [39mInside Main->handle_error | app.errorhandler( Exception )
[2016-01-05 21:04:57,665] {/Users/roporter/Desktop/Development/Python/myAPI/app/classes/requests.py->after_request:35} INFO - [97mINFO: [39mInside Main->returnResponse | app.after_request
[2016-01-11 07:54:32,432] {main.py-><module>:135} INFO - 



******************[93m2016-01-11 07:54:32.432633[39m*******************
[2016-01-11 07:54:33,147] {main.py-><module>:135} INFO - 



******************[93m2016-01-11 07:54:33.147155[39m*******************
[2016-01-11 07:54:36,021] {/Users/roporter/Desktop/Development/Python/myAPI/app/classes/requests.py->after_request:35} INFO - [97mINFO: [39mInside Main->returnResponse | app.after_request
[2016-01-11 07:54:36,530] {main.py->handle_error:54} INFO - [97mINFO: [39mInside Main->handle_error | app.errorhandler( Exception )
[2016-01-11 07:54:36,532] {/Users/roporter/Desktop/Development/Python/myAPI/app/classes/requests.py->after_request:35} INFO - [97mINFO: [39mInside Main->returnResponse | app.after_request
[2016-01-11 07:54:36,808] {main.py->handle_error:54} INFO - [97mINFO: [39mInside Main->handle_error | app.errorhandler( Exception )
[2016-01-11 07:54:36,811] {/Users/roporter/Desktop/Development/Python/myAPI/app/classes/requests.py->after_request:35} INFO - [97mINFO: [39mInside Main->returnResponse | app.after_request
[2016-01-11 07:54:36,934] {main.py->handle_error:54} INFO - [97mINFO: [39mInside Main->handle_error | app.errorhandler( Exception )
[2016-01-11 07:54:36,936] {/Users/roporter/Desktop/Development/Python/myAPI/app/classes/requests.py->after_request:35} INFO - [97mINFO: [39mInside Main->returnResponse | app.after_request
[2016-01-11 07:54:48,359] {/Users/roporter/Desktop/Development/Python/myAPI/app/classes/requests.py->after_request:35} INFO - [97mINFO: [39mInside Main->returnResponse | app.after_request
[2016-01-11 07:54:48,955] {main.py->handle_error:54} INFO - [97mINFO: [39mInside Main->handle_error | app.errorhandler( Exception )
[2016-01-11 07:54:48,956] {/Users/roporter/Desktop/Development/Python/myAPI/app/classes/requests.py->after_request:35} INFO - [97mINFO: [39mInside Main->returnResponse | app.after_request
[2016-01-11 07:54:49,037] {main.py->handle_error:54} INFO - [97mINFO: [39mInside Main->handle_error | app.errorhandler( Exception )
[2016-01-11 07:54:49,038] {/Users/roporter/Desktop/Development/Python/myAPI/app/classes/requests.py->after_request:35} INFO - [97mINFO: [39mInside Main->returnResponse | app.after_request
[2016-01-11 07:54:49,227] {main.py->handle_error:54} INFO - [97mINFO: [39mInside Main->handle_error | app.errorhandler( Exception )
[2016-01-11 07:54:49,229] {/Users/roporter/Desktop/Development/Python/myAPI/app/classes/requests.py->after_request:35} INFO - [97mINFO: [39mInside Main->returnResponse | app.after_request
[2016-01-11 07:54:55,757] {/Users/roporter/Desktop/Development/Python/myAPI/app/classes/requests.py->after_request:35} INFO - [97mINFO: [39mInside Main->returnResponse | app.after_request
[2016-01-11 07:54:56,428] {main.py->handle_error:54} INFO - [97mINFO: [39mInside Main->handle_error | app.errorhandler( Exception )
[2016-01-11 07:54:56,429] {/Users/roporter/Desktop/Development/Python/myAPI/app/classes/requests.py->after_request:35} INFO - [97mINFO: [39mInside Main->returnResponse | app.after_request
[2016-01-11 07:54:56,587] {main.py->handle_error:54} INFO - [97mINFO: [39mInside Main->handle_error | app.errorhandler( Exception )
[2016-01-11 07:54:56,589] {/Users/roporter/Desktop/Development/Python/myAPI/app/classes/requests.py->after_request:35} INFO - [97mINFO: [39mInside Main->returnResponse | app.after_request
[2016-01-11 07:54:56,777] {main.py->handle_error:54} INFO - [97mINFO: [39mInside Main->handle_error | app.errorhandler( Exception )
[2016-01-11 07:54:56,780] {/Users/roporter/Desktop/Development/Python/myAPI/app/classes/requests.py->after_request:35} INFO - [97mINFO: [39mInside Main->returnResponse | app.after_request
[2016-01-11 07:55:06,310] {/Users/roporter/Desktop/Development/Python/myAPI/app/classes/requests.py->after_request:35} INFO - [97mINFO: [39mInside Main->returnResponse | app.after_request
[2016-01-11 07:55:06,749] {main.py->handle_error:54} INFO - [97mINFO: [39mInside Main->handle_error | app.errorhandler( Exception )
[2016-01-11 07:55:06,750] {/Users/roporter/Desktop/Development/Python/myAPI/app/classes/requests.py->after_request:35} INFO - [97mINFO: [39mInside Main->returnResponse | app.after_request
[2016-01-11 07:55:06,848] {main.py->handle_error:54} INFO - [97mINFO: [39mInside Main->handle_error | app.errorhandler( Exception )
[2016-01-11 07:55:06,850] {/Users/roporter/Desktop/Development/Python/myAPI/app/classes/requests.py->after_request:35} INFO - [97mINFO: [39mInside Main->returnResponse | app.after_request
[2016-01-11 07:55:06,937] {main.py->handle_error:54} INFO - [97mINFO: [39mInside Main->handle_error | app.errorhandler( Exception )
[2016-01-11 07:55:06,940] {/Users/roporter/Desktop/Development/Python/myAPI/app/classes/requests.py->after_request:35} INFO - [97mINFO: [39mInside Main->returnResponse | app.after_request
[2016-01-11 07:55:16,041] {/Users/roporter/Desktop/Development/Python/myAPI/app/classes/requests.py->after_request:35} INFO - [97mINFO: [39mInside Main->returnResponse | app.after_request
[2016-01-11 07:55:16,531] {main.py->handle_error:54} INFO - [97mINFO: [39mInside Main->handle_error | app.errorhandler( Exception )
[2016-01-11 07:55:16,532] {/Users/roporter/Desktop/Development/Python/myAPI/app/classes/requests.py->after_request:35} INFO - [97mINFO: [39mInside Main->returnResponse | app.after_request
[2016-01-11 07:55:16,657] {main.py->handle_error:54} INFO - [97mINFO: [39mInside Main->handle_error | app.errorhandler( Exception )
[2016-01-11 07:55:16,658] {/Users/roporter/Desktop/Development/Python/myAPI/app/classes/requests.py->after_request:35} INFO - [97mINFO: [39mInside Main->returnResponse | app.after_request
[2016-01-11 07:55:16,806] {main.py->handle_error:54} INFO - [97mINFO: [39mInside Main->handle_error | app.errorhandler( Exception )
[2016-01-11 07:55:16,812] {/Users/roporter/Desktop/Development/Python/myAPI/app/classes/requests.py->after_request:35} INFO - [97mINFO: [39mInside Main->returnResponse | app.after_request
[2016-01-11 07:56:52,722] {/Users/roporter/Desktop/Development/Python/myAPI/app/classes/requests.py->after_request:35} INFO - [97mINFO: [39mInside Main->returnResponse | app.after_request
[2016-01-11 07:56:53,164] {main.py->handle_error:54} INFO - [97mINFO: [39mInside Main->handle_error | app.errorhandler( Exception )
[2016-01-11 07:56:53,166] {/Users/roporter/Desktop/Development/Python/myAPI/app/classes/requests.py->after_request:35} INFO - [97mINFO: [39mInside Main->returnResponse | app.after_request
[2016-01-11 07:56:53,318] {main.py->handle_error:54} INFO - [97mINFO: [39mInside Main->handle_error | app.errorhandler( Exception )
[2016-01-11 07:56:53,319] {/Users/roporter/Desktop/Development/Python/myAPI/app/classes/requests.py->after_request:35} INFO - [97mINFO: [39mInside Main->returnResponse | app.after_request
[2016-01-11 07:56:53,397] {main.py->handle_error:54} INFO - [97mINFO: [39mInside Main->handle_error | app.errorhandler( Exception )
[2016-01-11 07:56:53,398] {/Users/roporter/Desktop/Development/Python/myAPI/app/classes/requests.py->after_request:35} INFO - [97mINFO: [39mInside Main->returnResponse | app.after_request
[2016-01-11 07:57:00,788] {/Users/roporter/Desktop/Development/Python/myAPI/app/classes/requests.py->after_request:35} INFO - [97mINFO: [39mInside Main->returnResponse | app.after_request
[2016-01-11 07:57:01,345] {main.py->handle_error:54} INFO - [97mINFO: [39mInside Main->handle_error | app.errorhandler( Exception )
[2016-01-11 07:57:01,346] {/Users/roporter/Desktop/Development/Python/myAPI/app/classes/requests.py->after_request:35} INFO - [97mINFO: [39mInside Main->returnResponse | app.after_request
[2016-01-11 07:57:01,487] {main.py->handle_error:54} INFO - [97mINFO: [39mInside Main->handle_error | app.errorhandler( Exception )
